### PR TITLE
[INLONG-8354][Manager] Support previewing data of TubeMQ

### LINKED ISSUE #8354: [Improve][Manager] Support previewing data of TubeMQ
### Description

Support previewing data of tube.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #8354 

### Motivation

Support previewing data of tube.

### Modifications

Support previewing data of tube.

### CODE REVIEW

**Review by gosonzhang — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQResourceOperator.java [hunk 1]
```diff
@@ -108,4 +112,25 @@ public void deleteQueueForStream(InlongGroupInfo groupInfo, InlongStreamInfo str
...
 
+    public List<BriefMQMessage> queryLastestMessage(InlongGroupInfo groupInfo, InlongStreamInfo streamInfo,
+            Integer messageCount) {
+        Preconditions.expectNotNull(groupInfo, "inlong group info cannot be null");
+
+        String groupId = groupInfo.getInlongGroupId();
+        List<BriefMQMessage> briefMQMessages = new ArrayList<>();
+
+        try {
+            // 1. create tubemq topic
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQResourceOperator.java:123]: create topic here?

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQOperator.java [hunk 2]
```diff
@@ -248,4 +265,60 @@ private void createConsumerGroupOpt(String masterUrl, String topicName, String c
...
+            Integer messageCount, InlongStreamInfo streamInfo) {
+        LOGGER.info("begin to query message for topic {}", topicName);
+        String masterUrl = tubeCluster.getMasterWebUrl();
+        TubeBrokerInfo brokerView = this.getBrokerInfo(masterUrl);
+        String brokerUrl = brokerView.getOneWorkBrokerIp();
+
+        List<BriefMQMessage> messageList = new ArrayList<>();
+        try {
+            String url =
+                    "http://" + brokerUrl + QUERY_LAST_MESSAGE_PATH + TOPIC_NAME + topicName + MSG_COUNT + messageCount;
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQOperator.java:281]: It is recommended to provide a template, and then use the `String.format()` method to observe the URL format more directly.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQOperator.java [hunk 3]
```diff
@@ -248,4 +265,60 @@ private void createConsumerGroupOpt(String masterUrl, String topicName, String c
...
+                String msg = String.format("failed to query message for topic %s, error: %s",
+                        topicName, response.getErrMsg());
+                LOGGER.error(msg + " in {} for broker {}", masterUrl, brokerUrl);
+                throw new BusinessException(msg);
+            }
+            int index = 0;
+            for (TubeDataInfo tubeDataInfo : response.getDataSet()) {
+                Map<String, String> map = new HashMap<>();
+                for (String kv : tubeDataInfo.getAttr().split(InlongConstants.COMMA)) {
+                    map.put(kv.split("=")[0], kv.split("=")[1]);
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQOperator.java:304]: Please extract the `=` into `InlongConstants` class.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java [hunk 4]
```diff
@@ -187,4 +187,6 @@ public class InlongConstants {
      */
     public static final String BATCH_PARSING_FILED_JSON_COMMENT_PROP = "desc";
 
+    public static final String MSG_ENCODE_VER = "msgEnType";
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java:190]: Add some docs for this constant, thx.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQResourceOperator.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQResourceOperator.java]: Done.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQOperator.java (hunk 3 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/tubemq/TubeMQOperator.java]: Done.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java (hunk 4 above)
Comment (fuweng11) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java:195]: Done.

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**